### PR TITLE
[test/plugin] fix archetypes

### PR DESCRIPTION
**Pull request by arugal**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues
#3583

### CODE REVIEW

**Comment by arugal** [comment]
image
FYI @dmsolr

File: test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -16,15 +16,15 @@
   ~ limitations under the License.
   ~
   -->
-<Configuration status="WARN">
+<Configuration status="info">
     <Appenders>
-        <Console name="Console" target="SYSTEM_ERR">
-            <PatternLayout charset="UTF-8" pattern="[%d{yyyy-MM-dd HH:mm:ss:SSS}] [%p] - %l - %m%n"/>
+        <Console name="Console" target="SYSTEM_OUT">
```
Comment (dmsolr) [inline test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml]: Should be System.err here

File: test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml [hunk 2]
```diff
@@ -16,15 +16,15 @@
...
+            <PatternLayout pattern="%d [%traceId] %-5p %c{1}:%L - %m%n"/>
         </Console>
     </Appenders>
     <Loggers>
-        <Root level="WARN">
+        <Root level="info">
```
Comment (dmsolr) [inline test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml]: I think it is higher level better. this level will print many messages.

Comment (arugal) [reply test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml]: done

File: test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml (hunk 1 above)
Comment (arugal) [reply test/plugin/archetypes/jvm/src/main/resources/archetype-resources/src/main/resources/log4j2.xml]: done

**Pull request merged**